curriculum/challenges/english/25-front-end-development/workshop-caesar-cipher/681a2a6cf95e82ca30866364.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 681a2a6cf95e82ca30866364
+title: Step 23
+challengeType: 20
+dashedName: step-23
+---
+
+# --description--
+
+Declare two functions named `encrypt` and `decrypt`, both with `text` and `shift` parameters.
+
+You'll use `encrypt` for the encryption process, and `decrypt` for the decryption, labeling the two actions with a descriptive name.
+
+Return a `caesar` call passing in `text` and `shift` from both your new functions, but make sure to pass in also `False` as the third argument for the `decrypt` function.
+
+# --hints--
+
+You should have a function named `encrypt` with two parameters, `text` and `shift`.
+
+```js
+({ test: () => assert(runPython(`_Node(_code).find_function("encrypt").has_args("text, shift")`)) })
+```
+
+Your `encrypt` function should return `caesar(text, shift)`.
+
+```js
+({ test: () => assert(runPython(`_Node(_code).find_function("encrypt").has_return("caesar(text, shift)")`)) })
+```
+
+You should have a function named `decrypt` with two parameters, `text` and `shift`.
+
+```js
+({ test: () => assert(runPython(`_Node(_code).find_function("decrypt").has_args("text, shift")`)) })
+```
+
+Your `decrypt` function return `caesar(text, shift, False)`.
```

Comment:
```suggestion
Your `decrypt` function should return `caesar(text, shift, encrypt=False)`.
```

Reply:
```suggestion
Your `decrypt` function should return `caesar(text, shift, False)`.
```
It's not mandatory to use the keyword argument here. The test accepts both. But maybe we can add a step to enforce the use of it.

Reply:
If the test accepts both then it is fine.